<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ass_1</class>
 <widget class="QMainWindow" name="ass_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_42">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>请填写初次评估信息</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>931</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>再次评估信息</string>
    </property>
    <widget class="QLabel" name="label_43">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>54</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>肿胀：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_44">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>54</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>日期：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_45">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>病史/主诉：</string>
     </property>
    </widget>
    <widget class="QLabel" name="History_10">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>370</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_46">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>肌力：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_47">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>备注：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_48">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>10</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>关节活动度：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_49">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>70</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>疼痛程度：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_50">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>70</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>其他异常：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="date1">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="zhongzhang1">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="guanjie1">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>20</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="jili1">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="pain1">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>80</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="other1">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>80</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="beizhu1">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>140</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_51">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>如：20161110</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>931</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>基本信息</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>10</y>
       <width>54</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>姓名：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>54</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>编号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>病史/主诉：</string>
     </property>
    </widget>
    <widget class="QLabel" name="History_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>370</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>症状：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>70</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>诊断：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>131</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>项目及次数：：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_34">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>131</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>等线</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>治疗部位：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_40">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>181</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>注：如多个请用顿号隔开，如“脚、踝”</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_41">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>180</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>注：用“项目+次数”表示，如多个项目请用逗号隔开
例：超声波+10，超短波+11</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Number">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Name">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>20</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Zheng">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Zhenduan">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>80</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Buwei">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>141</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Xiangmu">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>141</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>522</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
